test/services/DateServiceSpec.scala

```diff
@@ -271,6 +271,68 @@ class DateServiceSpec extends SpecBase with ScalaCheckPropertyChecks with Genera
     }
   }
 
+  ".isDateOfFirstSaleDifferentToCommencementDate" - {
+
+    "must return true if the Date Of First Sale is a different date to the Commencement Date" in {
+      val dates: Gen[(LocalDate, LocalDate)] = for {
+        dateOfFirstSale  <- datesBetween(
+          LocalDate.of(2021, 7, 1),
+          LocalDate.now().minusDays(1)
+        )
+        extraMonths <- Gen.choose(2, 12)
+      } yield (dateOfFirstSale, dateOfFirstSale.plusMonths(extraMonths))
+
+      val commencementDate = LocalDate.now()
+
+      forAll(dates) {
+        case (dateOfFirstSale, today) =>
```

Comment:
This variable 'today' wont actually be today, it will be dateOfFirstSale.plusMonths(extraMonths).

Just pointing that out incase i _should_ be 'today' for the test. If not we can rename the variable

Reply:
Variable renamed